### PR TITLE
[yargs] fix: add a v15 folder without strictOptions

### PR DESCRIPTION
**Pull request by forivall**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/yargs/yargs/blob/v16.0.0/CHANGELOG.md#1600-2020-09-09 - notably, `strictOptions` _does not exist_ in yargs v15. it was added in yargs/yargs#1738
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

See also: #48657 which added strictOptions typings, erroneously to v15

### CODE REVIEW

File: types/yargs/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for yargs 15.0
+// Type definitions for yargs 16.0
 // Project: https://github.com/chevex/yargs, https://yargs.js.org
```
Comment (sheetalkamat) [inline types/yargs/index.d.ts:2]: There is no change between v16 and v15 currently. This change needs to be hold off if there is no change between these two versions

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/yargs/index.d.ts (hunk 1 above)
Comment (forivall) [reply types/yargs/index.d.ts:2]: `strictOptions` [doesn't exist in v15](https://github.com/yargs/yargs/compare/v15.4.2...v16.0.0#diff-452e4f3371a75b323f89da8de1a1707b1532abef67be1a02fe7e8396e0f35020R955). That's the point of this change.

Comment (sheetalkamat) [reply types/yargs/index.d.ts:2]: I dont see any change in v16? Can you point where the change is between v15 and v16 in these definitions?

Comment (forivall) [reply types/yargs/index.d.ts:2]: `.strictOptions` [exists here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/72cb30219ded5aa79b9fa554463f8f860e2e0f08/types/yargs/index.d.ts#L579) and i've [removed it here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/72cb30219ded5aa79b9fa554463f8f860e2e0f08/types/yargs/v15/index.d.ts#L573). See the [diff](https://diffy.org/diff/nngnb82nreufkht6njdc84cxr)

Comment (sheetalkamat) [reply types/yargs/index.d.ts:2]: Thank you. it got lost in big copy.. normally when versions change the change is in index.d.ts and not the older version. Thank you for the pointers.

**Pull request merged**